.github/workflows/towerfulltest.yml

```diff
@@ -0,0 +1,27 @@
+name: Tower AWS full size tests
```

Comment:
I'd rename these files to `tower_full_test_*`

Reply:
I modeled these after what nf-core pipelines use:
"name: nf-core AWS full size tests"
https://github.com/nf-core/taxprofiler/blob/dev/.github/workflows/awsfulltest.yml#L1